atlp-rwanda/commanders-rn-medica
# Ch GitHub actions setup

#### What does this PR do?

This pr creates github actions that build,  run tests and deploys the app  to expo on every pull request

#### Description of Task to be completed?

- [x] Create the workflow yml file
- [x]  Setup the build dependences in the  project 
- [x]  Include the build, test and deploy jobs in the yml file 

#### How should this be manually tested?

Navigate to   github actions in a workflow called medica and  see the jobs (build, test, deploy) that are running there 
you can test in your compiuter locally by learning npm run build to deploy the app 


#### What are the relevant pivotal tracker/Trello stories?

https://github.com/atlp-rwanda/commanders-rn-medica/issues/4

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
